includes/webserv.hpp

```diff
@@ -0,0 +1,69 @@
+#ifndef WEBSERV_H
+#define WEBSERV_H
+
+#include <arpa/inet.h>
+#include <cstdlib>
+#include <errno.h>
+#include <fcntl.h>
+#include <fstream>
+#include <iostream>
+#include <netinet/in.h>
+#include <ostream>
+#include <poll.h>
+#include <sstream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <vector>
```

Comment:
The header includes should be moved from webserv.hpp to the files where they are being properly used